customize_image.sh

```diff
@@ -314,27 +312,19 @@ if [ -e "$IMAGEDIR/etc/init.d/apache2" ] ; then
 fi
 
 if [ "$MINIMG" != "1" ]; then
-	# fake a revpi procfs ... otherwise NodeRed installer will get confused
-	for procfile in meminfo cpuinfo; do
-		touch "$IMAGEDIR/proc/$procfile"
-		mount -o ro,bind "$BAKERYDIR/templates/fake-procfs/$procfile" "$IMAGEDIR/proc/$procfile"
-	done
-
-	# install nodejs and nodered with an install script and revpi-nodes from npm repository
-	NODEREDSCRIPT="/tmp/update-nodejs-and-nodered.sh"
-	/usr/bin/curl -sL \
-		https://raw.githubusercontent.com/node-red/linux-installers/master/deb/update-nodejs-and-nodered\
-		--output "$IMAGEDIR/$NODEREDSCRIPT"
-	chmod 755 "$IMAGEDIR/$NODEREDSCRIPT"
-	NODERED_VER="3.0.2"
```

Comment:
Please use git revert for 29851fce0f3a8238ff69af11a643cd266e301142, this will allow us to remove all reverted commits from the bullseye branch before we merge it to master.

Reply:
I'd rather continue using the variable now. It remains.